.github/workflows/main.yml

```diff
@@ -0,0 +1,31 @@
+name: tests
+on:
+  push:
+  pull_request:
+    branches: [master]
+  schedule:
+    - cron: '0 0 * * 1'
+
+jobs:
+  tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        nimversion:
+        - stable
+        - devel
+        os:
+        - ubuntu-latest
+        - macOS-latest
+        - windows-latest
```

Comment:
`httpbeast` doesn't support windows. The windows test line should be removed.

Reply:
Thanks, fixed!